agent.py

```diff
@@ -50,11 +54,21 @@
 from utils.service_discovery.sd_backend import get_sd_backend
 
 # Constants
+from jmxfetch import JMX_CHECKS
 PID_NAME = "dd-agent"
 PID_DIR = None
 WATCHDOG_MULTIPLIER = 10
 RESTART_INTERVAL = 4 * 24 * 60 * 60  # Defaults to 4 days
 
+JMX_SUPERVISOR_ENTRY = 'datadog-agent:jmxfetch'
+JMX_GRACE_SECS = 2
+SERVICE_DISCOVERY_PREFIX = 'SD-'
+SD_PIPE_NAME = "dd-service_discovery"
+SD_PIPE_UNIX_PATH = "/tmp"
```

Comment:
Named pipes are only supported on windows (and I don't think all versions apply) in a special path, so it doesn't really apply to windows.

I used `/tmp` to avoid issues with the source installs.